src/game/Box.js

```diff
@@ -34,7 +33,7 @@ export default class Box extends Component {
 }
 
 const initImagesRequire = () => {
-    var pieceImageArray = [];
+    const pieceImageArray = [];
```

Comment:
You could have initialized every images directly here so you don't have to write those pesky indexes

```js
const pieceImageArray = [
    require('../../resources/images/pieceImage1.png'),
    ...
]
```